rust/agama-cli/src/commands.rs

```diff
@@ -21,7 +21,14 @@ pub enum Commands {
     /// Autoinstallation profile handling
     #[command(subcommand)]
     Profile(ProfileCommands),
-    /// Questions handling
+    /// Configuration for questions that come from installer
+    ///
+    /// Questions are raised when unexpected situation happen in Installer
```

Comment:
```suggestion
    /// Questions are raised when an unexpected situation happens in the installer:
```

Reply:
Well *unexpected* is wrong. It is expected, that's why we have coded a special case for it

Reply:
yeah, I am not sure what is better word. Maybe "uncommon" or "rare"? not sure